storagenode/piecestore/endpoint.go

```diff
@@ -98,6 +101,48 @@ type Endpoint struct {
 	liveRequests int32
 }
 
+// speedEstimation monitors state of incoming traffic. It would signal slow-speed
```

Comment:
nit: Usually it's slightly nicer when the utility structs / funcs are at the end. I.e. order by importance (mostly).

Reply:
@egonelbre for the sake of perfection, I did the modification.